src/services/pool/calculator/calculator.sevice.ts

```diff
@@ -249,7 +245,7 @@ export default class CalculatorService {
   }
 
   public get isComposableStableLikePool(): boolean {
-    return isComposableStableLike(this.pool.value.poolType);
+    return isDeep(this.pool.value);
```

Comment:
This shouldn't have been changed, please revert:
```suggestion
    return isComposableStableLike(this.pool.value.poolType);
```